metricflow/sql_clients/doris.py

```
@@ -0,0 +1,173 @@
+                        values.append(f"'{escaped_value}'")
+                    elif isinstance(value, bool):
+                        values.append("1" if value else "0")
+                    elif isinstance(value, (int, float)):
+                        values.append(str(value))
+                    else:
+                        escaped_value = str(value).replace("'", "''")
+                        values.append(f"'{escaped_value}'")
+                values_list.append(f"({', '.join(values)})")
+
+            if values_list:
+                self.execute(f"INSERT INTO {sql_table.sql} VALUES {', '.join(values_list)}")
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)